cli/tests/testdata/test/interval.ts

```diff
@@ -0,0 +1 @@
+setInterval(function () {}, 0);
```

Comment:
Why does it exit instead of blocking forever?

Reply:
We only wait for the tests to complete, so the exit condition is when all tests complete rather than when all ops complete.